### PR TITLE
drop legacy sypport (fix #19); support verdaccio v6

### LINKED ISSUE #19: Its look like that legacy mode not working since verdaccio v5.22.0
[verdaccio v5.22.0](https://github.com/verdaccio/verdaccio/releases/tag/v5.22.0) came with some breaking changes in **legacy** tokens. Also legacy mode is enabled by default (hmm?). 

Note: JWT tokens work well.

### PR DESCRIPTION
**Pull request by dafanasiev**
subj

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -5,26 +5,37 @@ plugins: /verdaccio/plugins
...
+    # Optional id_token claim that will be used for username
     usernameClaim: preferred_username
 
-    # optional roles claim (id_token).
-    # claim can be array or string (comma-separated)
+    # Optional switch to alternative login method, using access token as password.
+    # The username must be the same as the one used to acquire the access token,
+    # the password must be an OIDC access token. The e-mail address will be ignored.
+    # If not set, the regular authentication flow will be used.
     rolesClaim: roles
```
Comment (aikoven) [inline config.yaml]: The comment is not related to `rolesClaim` field

**Review by aikoven — COMMENTED** [review]
Sorry for the late response and thank you for your contribution.

This generally looks good, please address the small issue below and I will merge it.

File: config.yaml (hunk 1 above)
Comment (dafanasiev) [reply config.yaml]: FIXED

**Comment by aikoven** [comment]
Thanks! Published `verdaccio-openid-connect@3.0.0`

**Pull request merged**